### PR TITLE
Review edits to guidelines.md

### PR DESCRIPTION
**Pull request by peknis01**

### CODE REVIEW

File: docs/reference/contributing/guidelines/guidelines.md [hunk 1]
```diff
@@ -8,60 +8,63 @@ To encourage collaboration, as well as robust, consistent and maintainable code,
...
 
 - Apache 2.0.
 - BSD 3-Clause.
 - MIT.
 
-For new Mbed projects, we suggest adopting the Apache 2.0 license. Note that any Mbed software release under a permissive license cannot accept any code that is licensed under a "copyleft" license. Doing so would prevent us from continuing to distribute our code under the permissive license. You are welcome to use Mbed software with copyleft licenses, as long as the rules of the copyleft license are followed.
+For new Mbed projects, we suggest adopting the Apache 2.0 license.
 
-You can find a more detailed description on licenses in the [contributing guidelines](../contributing/license.html).
+<span class="notes">**Note:** An Mbed software release under a permissive license cannot accept any code that is licensed under a "copyleft" license. Doing so would prevent us from distributing our code under the permissive license. You are welcome to use Mbed software with copyleft licenses, as long as the rules of the copyleft license are followed.</span>
```
Comment (melwee01) [inline docs/reference/contributing/guidelines/guidelines.md]: as long as you follow the rules?

File: docs/reference/contributing/guidelines/guidelines.md [hunk 2]
```diff
@@ -8,60 +8,63 @@ To encourage collaboration, as well as robust, consistent and maintainable code,
...
+
+Please, see the [contributing guidelines](../contributing/license.html) for a more detailed description on licenses.
```
Comment (melwee01) [inline docs/reference/contributing/guidelines/guidelines.md]:
```suggestion
Please, see the [contributing guidelines](../contributing/license.html) for a more detailed description of licenses.
```

File: docs/reference/contributing/guidelines/guidelines.md [hunk 3]
```diff
@@ -8,60 +8,63 @@ To encourage collaboration, as well as robust, consistent and maintainable code,
...
+You can request to become an organization member, but you must first ensure your GitHub profile meets the following requirements:
 
 - All users must have two-factor authentication enabled.
 - Arm staff must have their Name, Company (Arm) and Location publicly visible.
 - All others should have their Name and Company visible. Including your Location helps us interpret response times according to time zones.
 
 ### Leaving the ARMmbed organization
 
-When leaving the organization, please follow these [best practices](https://help.github.com/en/articles/best-practices-for-leaving-your-company).
+When leaving the organization, please follow the [best practices](https://help.github.com/en/articles/best-practices-for-leaving-your-company).
```
Comment (melwee01) [inline docs/reference/contributing/guidelines/guidelines.md]:
```suggestion
When leaving the organization, please follow [best practices](https://help.github.com/en/articles/best-practices-for-leaving-your-company).
```

File: docs/reference/contributing/guidelines/guidelines.md [hunk 4]
```diff
@@ -8,60 +8,63 @@ To encourage collaboration, as well as robust, consistent and maintainable code,
 
 ## How to contribute  
 
-Mbed OS has a team of maintainers who provide guidance and direction to contributors. This team is responsible for helping you get your changes in, as well as controlling the overall quality and consistency of the software. For more information about the maintainers and their responsibilities, please see [our workflow](https://os.mbed.com/docs/development/contributing/workflow.html#mbed-os-maintainers).
+Mbed OS has a team of maintainers who provide guidance and direction to contributors. This team is responsible for helping you get your changes in, as well as controlling the overall quality and consistency of the software. 
+<!-- I left out the first link, as we have the same link in the next paragraph -->
```
Comment (AnotherButler) [inline docs/reference/contributing/guidelines/guidelines.md]: 👍

**Review by AnotherButler — COMMENTED** [review]
Thanks for the PR 👍 It looks great.

**Review by AnotherButler — APPROVED** [review]

**Pull request merged**